lib/src/components/NxLoadWrapper/__tests__/NxLoadWrapper.test.tsx

```diff
@@ -4,48 +4,87 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxLoadWrapper, { Props } from '../NxLoadWrapper';
-import NxLoadError from '../../NxLoadError/NxLoadError';
-import NxLoadingSpinner from '../../NxLoadingSpinner/NxLoadingSpinner';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+import NxLoadWrapper from '../NxLoadWrapper';
 
 describe('NxLoadError', function() {
   const children = <div className="children"/>,
-      retryHandler = jest.fn(),
-      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxLoadWrapper, { children, retryHandler });
+      retryHandler = jest.fn();
+  const renderEl = rtlRenderElement(NxLoadWrapper, {children, retryHandler});
+  const quickRender = rtlRender(NxLoadWrapper, {children, retryHandler});
 
   it('renders a NxLoadError if there is an error', function() {
-    const ErrorFixture = () => <NxLoadError error="foo" />;
-
-    expect(getShallowComponent({ error: 'foo' })).toMatchElement(<ErrorFixture/>);
-    expect(getShallowComponent({ error: 'foo', loading: true })).toMatchElement(<ErrorFixture />);
+    const componentWithoutError = renderEl();
```

Comment:
The "without error" case is covered in the 'renders provided children if loading is false and error is unset' test, you don't need to check it here

Reply:
> The "without error" case is covered in the 'renders provided children if loading is false and error is unset' test, you don't need to check it here

Done ed0e4ce83719ef48e309b5fcd5736a055e8b224f